src/App.js

```diff
@@ -1,20 +1,26 @@
 import React from 'react'
 import { BrowserRouter as Router } from 'react-router-dom'
 import { ViewportProvider } from 'use-viewport'
+import { AppDataProvider } from './providers/AppData'
+import { OrgAppsProvider } from './providers/OrgApps'
 import { ConnectProvider as Connect } from './providers/Connect'
 import MainView from './components/MainView'
 import Routes from './Routes'
 
 function App() {
   return (
     <Connect>
-      <ViewportProvider>
-        <Router>
-          <MainView>
-            <Routes />
-          </MainView>
-        </Router>
-      </ViewportProvider>
+      <OrgAppsProvider>
+        <AppDataProvider>
```

Comment:
In the future we will remove `AppDataProvider` and move the requests into smaller to progressively request things in views as we need. I still think it is a nice idea to provide the base apps via `OrgAppsProvider` though.